### PR TITLE
[videoFilter] New filter: 3D LUT

### PR DESCRIPTION
**Pull request by szlldm**
Currently HaldCLUTs and .cube files are supported.
For testing: https://gmic.eu/color_presets/

### CODE REVIEW

File: avidemux_plugins/ADM_videoFilters6/cubicLUT/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+# /QT4
+
+
+
+#------------ Cli Version ----------------
+INCLUDE(vf_plugin_cli)
+SET(CubicLUTCli_SRCS cli/DIA_cubicLUT.cpp)
+IF(DO_CLI)
+    INIT_VIDEO_FILTER_CLI(  ADM_vf_cubicLUTCli ${CubicLUTCli_SRCS} ${CubicLUTCommon_SRCS})
+    TARGET_LINK_LIBRARIES(ADM_vf_cubicLUTCli ADM_coreImageLoader6)
```
Comment (eumagga0x2a) [inline avidemux_plugins/ADM_videoFilters6/cubicLUT/CMakeLists.txt:32]: The CLI version probably needs to be linked against ADM_coreVideoCodec6 too, else the build fails.

File: avidemux_plugins/ADM_videoFilters6/cubicLUT/CMakeLists.txt [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    ADM_LINK_THREAD(ADM_vf_cubicLUT${QT_LIBRARY_EXTENSION})
```
Comment (eumagga0x2a) [inline avidemux_plugins/ADM_videoFilters6/cubicLUT/CMakeLists.txt:33]: This was likely meant to be `ADM_LINK_THREAD(ADM_vf_cubicLUTCli)`, necessary to fix the build.

File: avidemux_plugins/ADM_videoFilters6/cubicLUT/CMakeLists.txt (hunk 1 above)
Comment (szlldm) [reply avidemux_plugins/ADM_videoFilters6/cubicLUT/CMakeLists.txt]: thx, should be fixed

Comment (szlldm) [reply avidemux_plugins/ADM_videoFilters6/cubicLUT/CMakeLists.txt]: thx, should be fixed

File: avidemux_plugins/ADM_videoFilters6/cubicLUT/CMakeLists.txt (hunk 2 above)
Comment (szlldm) [reply avidemux_plugins/ADM_videoFilters6/cubicLUT/CMakeLists.txt]: thx, should be fixed

**Comment by eumagga0x2a** [comment]
Thanks!

**Pull request merged**